other/maven-google-appengine-standard/pom.xml

```diff
@@ -11,8 +11,8 @@
     <artifactId>maven-google-appengine-standard</artifactId>
 
     <properties>
-        <kotlin.version>1.3.61</kotlin.version>
-        <ktor.version>1.3.0</ktor.version>
+        <kotlin.version></kotlin.version>
```

Comment:
This is not good since users copy-paste this and may be confused

Reply:
It's only for one purpose: fail if no CLI argument. There was a problem here: no CLI arguments were provided, and the samples always use old versions. 

Maybe we should add XML comment here